pkg/engine/crds.go

```diff
@@ -41,6 +41,10 @@ func Ensure(config *rest.Config) error {
 	if err != nil && !errors.IsAlreadyExists(err) {
 		return fmt.Errorf("error creating the Endpoint CRD: %s", err)
 	}
+	_, err = apiext.ApiextensionsV1beta1().CustomResourceDefinitions().Create(getMcsCRD())
```

Comment:
Why is this here? MCS is not required for engine. Shouldn't this be in pkg/lighthouse/crds.go?

Reply:
Done.